CuratedContent/WorkingEffectivelyWithLegacyCode.md

```diff
@@ -0,0 +1,34 @@
+# Working Effectively with Legacy Code
+
+## Feathers, Michael, 2005
+
+This is really the book that gives one hope that we can take very bad software
```

Comment:
Not sure we wanna use the word "bad" here. Maybe a phrase "software written with obsolete development processes"

Reply:
Yes, how about "messy" or "hard to work with" or "hard to maintain"?

Reply:
@markcmiller86, please take a look at my latest commit 4b0fca2 and see if this is better.

Also note that I switched to the one-line-per-sentence format which massively improves concurrent editing and merging.

Reply:
I think any of those are better. You could get cute and coin a new phrase like "fossil-ware" or "vintage-ware" "bygone-ware" ;)